### PR TITLE
[ticket/15915] Add template events to posting_attach_body.html

### PR DESCRIPTION
**Pull request by rxu**
Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

<a href="https://tracker.phpbb.com/browse/PHPBB3-15915">PHPBB3-15915</a>.

### CODE REVIEW

**Comment by rxu** [comment]
!set Event

**Comment by 3D-I** [comment]
Any particular reason why not to use Twig syntax?

**Comment by marc1706** [comment]
As @3D-I mentioned, please use twig syntax. Especially when it's going against master.

File: phpBB/docs/events.md [hunk 1]
```diff
@@ -1595,6 +1595,34 @@ overall_header_stylesheets_after
...
+* Purpose: Add content after attachment row in the file list
+
+posting_attach_body_attach_row_before
+===
+* Locations:
+    + styles/prosilver/template/posting_attach_body.html
+* Since: 3.2.6-RC1
+* Purpose: Add content before attachment row in the file list
+
+posting_attach_body_attach_row_append
```
Comment (3D-I) [inline phpBB/docs/events.md]: `posting_attach_body_attach_row_append` 
should go right after
`posting_attach_body_attach_row_after`
if I am not mistaken.

Comment (rxu) [reply phpBB/docs/events.md]: yeah, I guess so.

**Comment by rxu** [comment]
I'm not sure about using Twig syntax here as there's no Twig syntax in the file at all.

**Pull request merged**